Firebase/Auth/Source/Backend/RPC/MultiFactor/SignIn/FIRStartMfaSignInRequest.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRAuthRPCRequest.h"
+#import "FIRIdentityToolkitRequest.h"
+#import "FIRAuthProtoStartMfaPhoneRequestInfo.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRStartMfaSignInRequest : FIRIdentityToolkitRequest <FIRAuthRPCRequest>
+
+@property(nonatomic, copy, readonly, nullable) NSString *mfaProvider;
+
+@property(nonatomic, copy, readonly, nullable) NSString *mfaPendingCredential;
+
+@property(nonatomic, copy, readonly, nullable) NSString *mfaEnrollmentID;
```

Comment:
The `MFA` should be capitalized here and in the initializer. Even if you can't do the type right now, the properties should be named appropriately.